lib/lndclient/LndClient.ts

```diff
@@ -2,29 +2,35 @@ import grpc, { Metadata } from 'grpc';
 import fs from 'fs';
 
 import Logger from '../Logger';
-import BaseClient, { ClientStatus } from'../BaseClient';
+import BaseClient, { ClientStatus } from '../BaseClient';
 import errors from './errors';
 
 type LndClientConfig = {
   disable: boolean;
   datadir: string;
+  host: string;
+  port: number;
   rpcprotopath: string;
 };
 
 /** A class representing a client to interact with a running lnd instance. */
-class LndClient extends BaseClient{
+class LndClient extends BaseClient {
   lightning: any;
   meta: Metadata | undefined;
+  port: number;
+  host: string;
```

Comment:
Good idea to make `host` and `port` for the lnd grpc service configurable, however I don't think we need to make them class properties. They're only used in the constructor to initialize the `lightning` property, so we can drop them here (but leave them in `LndClientConfig`).